javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/matcher/GlobalIgnoresMatcher.java

```diff
@@ -60,11 +60,14 @@ public boolean matches(T target) {
     if (name.startsWith("net.bytebuddy.")
         || name.startsWith("jdk.")
         || name.startsWith("org.aspectj.")
+        || name.startsWith("datadog.")
         || name.startsWith("com.intellij.rt.debugger.")
         || name.startsWith("com.p6spy.")
         || name.startsWith("com.dynatrace.")
         || name.startsWith("com.jloadtrace.")
         || name.startsWith("com.appdynamics.")
+        || name.startsWith("com.newrelic.")
```

Comment:
IIRC, NR couldn't have this here because there are tons of non-agent classes around at NR that still need instrumentation.

Reply:
do you propose to remove `com.newrelic.` from here or are those classes used only internally at NR?

Reply:
I don't have a proposal (I'm not at NR any more), but I was responsible for removing this originally, because NR wanted to be able to instrument their own code with the otel agent, and they couldn't because of this.

Reply:
Is there anybody from NewRelic to comment? 

cc) @breedx-nr

Maybe we could exclude some subpackages:
https://github.com/newrelic/newrelic-java-agent/tree/main/newrelic-agent/src/main/java/com/newrelic/agent
https://github.com/newrelic/newrelic-java-agent/tree/main/newrelic-api/src/main/java/com/newrelic/api/agent

Reply:
heh. @breedx-nr is no longer at New Relic, either. I'm not sure who the right person would be.

Reply:
maybe @jasonjkeller?

Reply:
I have changed `com.newrelic` to `com.newrelic.agent` and `com.newrelic.api.agent`. I doupt they use the OTEL agent internally though.